docs/users.md

```diff
@@ -0,0 +1,62 @@
+# Users
+
+A user as defined in identity-manager-sts is any unique subject as identified by an issuer. Users are bound to the issuer that provides identifying JWTs for them (i.e., `foo@example.com` from one OIDC provider is not the same user as `foo@example.com` from another OIDC provider).
+
+## Tenant scoping
+
+Currently, identity-manager-sts does not support any particular mechanism for binding an issuer to a given tenant. This work is being tracked as part of Issue [#23][issue-23].
+
+At minimum, it is expected that identity-manager-sts should ultimately support constraining tokens to tenants using the OAuth 2.0 `audience` parameter.
```

Comment:
The `audience` parameter could work. But, an alternative could be to use the `resource` parameter to specify the tenant/directory, and restrict the resulting `audience` to portray the given infratographer instance (URL).

What do you think?

Reply:
`resource` defines absolute URIs that map to `aud` claims in [RFC 9068][rfc-9068] access tokens (not sure why `resource` was chosen over `audience`). Are you suggesting we use `resource` to specify both the applicable tenant/directory and APIs?

[rfc-9068]: https://www.rfc-editor.org/rfc/rfc9068.html#section-3

Reply:
I was only referring to the applicable directory sub-tree. Did you have in mind specifying the target APIs as part of the resource then?